Makefile

```diff
@@ -158,6 +158,7 @@ release-prep:
 	@cp -r ./plugins release_deps/
 	@cp config/example.yaml release_deps/config.yaml
 	@cp config/logging.yaml release_deps/logging.yaml
+	@cp service/com.observiq.collector.plist release_deps/com.observiq.collector.plist
```

Comment:
Why is there the need to add this command? like why copy the service file into the release dep file for the sake of prepping for a release?

Reply:
This is a good question; @cpheps Do you remember why we implemented this pattern of copying to release_deps instead of referencing the original path directly in goreleaser? It makes sense for the stuff we download, not sure why we do this for plugins or config.yaml, actually.

Reply:
I think it was for downloading. Plugins use to be downloaded as well as the JMX jar. We probably don't need it now if there's a known static location for things.

Reply:
We also do renames in there looks like for the `config.yaml`

Reply:
I'll leave this as-is for right now to keep it consistent with the other artifacts; Might be something to revisit later.